helm/sdn-dashboard/README.md

```diff
@@ -0,0 +1,87 @@
+# sdn-dashboard
+
+-----------------------
+
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![AppVersion: 1.0.0](https://img.shields.io/badge/AppVersion-1.0.0-informational?style=flat-square) 
```

Comment:
add a hook to website

Reply:
Added moved docs to docs/installation/helm/sdn-dashboard.md and added hook to mkdocs.yml